primitives/src/currency.rs

```diff
@@ -184,6 +184,7 @@ create_currency_id! {
 		AUSD("Acala Dollar", 12) = 1,
 		DOT("Polkadot", 10) = 2,
 		LDOT("Liquid DOT", 10) = 3,
+		TAI("Taiga", 12) = 4,
 		// 20 - 39: External tokens (e.g. bridged)
```

Comment:
move this to 132, it is part of the Karura & Kusama token

Reply:
got it